src/js/polyfills/xhr.js

```diff
@@ -106,14 +106,15 @@ class XMLHttpRequest extends EventTarget {
             const path = globalThis[Symbol.for('tjs.internal.modules.path')];
             const TJS_HOME = tjs.env.TJS_HOME ?? path.join(tjs.homeDir, '.tjs');
 
-            this[kXHR].cookieJar = path.join(TJS_HOME, 'cookies');
+            tjs.makeDir(TJS_HOME, { recursive: true });
```

Comment:
This returns a promise, so it's not guaranteed to finish before your next statement. Plus, if feels wrong to do here.

Reply:
There is an internal mkdirSync function in storage.js you may want to move that one around to use it.